docs/source/usersguide/random_ray.rst

```diff
@@ -0,0 +1,303 @@
+
+--------
+Plotting
+--------
+
+Visualization of geometry is handled in the same way as normal with OpenMC (see :ref:`plotting guide <usersguide_plots>` for more details). I.e., ``openmc --plot`` is handled without any modifications, as the random ray solver uses the same geometry definition as in Monte Carlo.
+
+In addition to OpenMC's standard geometry plotting mode, the random ray solver also features an additional method of data visualization. If a ``plots.xml`` file is present, any voxel plots that are defined will be output at the end of a random ray simulation. Rather than being stored in HDF5 file format, the random ray plotting will generate ``.vtk`` files that can be directly read and plotted with `Paraview <https://www.paraview.org/>`_ (a free application). 
+
+In fixed source Monte Carlo (MC), by default the only thing we know after a simulation is the escape fraction. In a k-eigenvalue MC solve, by default all we know is the eigenvalue and escape fraction. Spatial flux information is left totally up to the user to record, and often fine-grained spatial meshes are considered costly/unnecessary, so it makes no sense in MC mode to try to attempt to plot any spatial flux or power info by default. Conversely, in random ray, the solver functions by estimating the multigroup source and flux spectrums in every fine-grained FSR each iteration. Thus, in random ray, in both fixed source and eigenvalue simulations, the simulation always finishes with a well converged flux estimate for all areas. As such, it is much more common in random ray, MOC, and other deterministic codes to plot in situ commonly as global spatial flux information is always available. In the future, all FSR data will be made available in the statepoint file, such that users will still have the ability to plot/manipulate it on the python end, although statepoint support is not yet available.
+
+Only voxel plots will be used to generate output -- other plot types present in the ``plots.xml`` file will be ignored. The following fields will be written to the VTK structured grid file:
+
+    - material
+    - FSR index
+    - flux spectrum (for each energy group)
+    - total fission source (integrated across all energy groups)
+
+------------------------------------------
+Inputting Multigroup Cross Sections (MGXS) 
+------------------------------------------
+
+Multigroup cross sections for use with OpenMC's random ray solver are input the same way as with OpenMC's traditional multigroup Monte Carlo mode. There is more information on generating multigroup cross sections via OpenMC in the :ref:`multigroup materials <create_mgxs>` user guide. A user may also wish to use an existing multigroup library. An example of using OpenMC's python interface to generate a correctly formatted ``mgxs.h5`` input file is given below, which defines a seven group cross section dataset.
```

Comment:
It's probably worth pointing out here that at the moment, only P0, isothermal, isotropic XS are supported.

Reply:
Good thinking! This has been called out as a note at the end of the section.